praw/models/reddit/emoji.py

```diff
@@ -0,0 +1,71 @@
+"""Provide the EMOJI class."""
+from ...const import API_PATH
+from .base import RedditBase
+
+
+class Emoji(RedditBase):
+    """An individual Emoji object."""
+
+    __hash__ = RedditBase.__hash__
+    STR_FIELD = 'name'
+
+    def __init__(self, reddit, subreddit, name, _data=None):
+        """Construct an instance of the Emoji object."""
+        self.reddit = reddit
+        self.subreddit = subreddit
+        self.name = name
+        super(Emoji, self).__init__(reddit, _data)
+
+    def add(self, filepath):
```

Comment:
I think the add method would make more sense on the `SubredditEmoji` class so that the interface is:

    subreddit.emoji.add('name', 'path')

What do you think? That _should_ be consistent with other add methods, right?